src/components/svg-wrapper.tsx

```diff
@@ -72,6 +73,12 @@ const SvgWrapper = () => {
             // only type is needed
             .forEach(([type]) => loadFontCss(type as NodeType));
     }, [refreshNodes]);
+    // Update all components that requires a bbox after fonts are loaded.
+    // bbox calculated before fonts are loaded will be incorrect.
+    const { update } = React.useContext(SvgAssetsContext);
+    React.useEffect(() => {
+        document.fonts.load('12px Arial', 'ABCDEFG123456').finally(() => setTimeout(update, 100));
```

Comment:
Generally speaking, we should test all the characters used, against the font that used for rendering. For alphanumeric characters, we could simply test A-Za-z0-9. For different station components, we should test different fonts.

Since the `update` function only works for GZMTR's station number currently, it's not necessary to change this line.

Reply:
Or just `document.fonts.load('12px Arial').finally(...)`? Also, will there be a difference for `load` if the font is in a single file instead of what we load MTR where fonts are split into multiple files? I mean if there is only a single file, test one character is the same as all others as either all glyphs are loaded or none.

Reply:
I’m not sure if we can omit the characters to be tested. For the MTR case, a **single** `document.load` with all required characters can handle all the font chunks. 

For GZMTR, the current implementation is sufficient for the use case.